lib/routes/routeBackbeat.js

```diff
@@ -120,6 +120,12 @@ function putMetadata(request, response, bucketInfo, objMd, log, callback) {
             // FIXME: add error type MalformedJSON
             return callback(errors.MalformedPOSTRequest);
         }
+        const { headers, bucketName, objectKey } = request;
+        // check if it's metadata only operation
+        if (headers['x-scal-replication-content'] === 'METADATA') {
+            // copy original data locations
+            omVal.location = Array.from(objMd.location);
```

Comment:
This seems to be occurring in case the object does not exist on the target. I think in such case we should just reject the whole request.